### PR TITLE
The dunce cap makes you a dunce (Updated)

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
<a href="https://www.patreon.com/barrytheman"> </a>
(Generously funded by 1$ on Patreon to make this PR)
The dunce cap will now make you say brain damage lines when worn.

:cl:
 * tweak: The dunce cap now makes you somewhat of a dunce.

### CODE REVIEW

**Comment by GreatBigFailure** [comment]
I see no reason not to. It's kind of funny. 
If people don't like it this way maybe make it apply the damage on equip but remove it when you unequip it.

**Comment by dunsel-hordon** [comment]
so will it check if you already have 20 brain damage? Also at 20 brain damage do you start doing retard things?

**Comment by dunsel-hordon** [comment]
also might want to have it in the item description that it gives a bit of brain damage but beyond that i approve

**Comment by TGFH** [comment]
Stop trying to make everyone as brain damaged as you.

**Comment by bathosbathtime** [comment]
I shouldn't have to worry about my actions being considered griff for putting a paper cone hat on someone else to make fun of them.

**Comment by boy2mantwicethefam** [comment]
@dunsel-hordon Aye, sure
@TGFH It's a commission. Do you even read the PR before you comment?
@bathosbathtime The brain damage is minimal specifically because of the cat girl ears shitstorm, 20 brain damage is nothing and all the debilitation from brain damage only happens at 60 and above brain damage

**Comment by dunsel-hordon** [comment]
Plus the side effects go away slight as they are after it's taken off

**Comment by jknpj** [comment]
This again?

**Comment by bathosbathtime** [comment]
I don't want to worry about any mechanical effects, no matter how minor, when putting a hat on someone to mock them. It is a dumb change that makes the act far more mean-spirited.

**Comment by boy2mantwicethefam** [comment]
As if putting the dunce cap on them would be far more mean-spirited if it dealt insignificant brain damage than it is now.
A rare hat too to boot.

**Comment by dunsel-hordon** [comment]
Not only is it a rare hat it still does basically nothing. It gives just enough brain damage to make the message appear that ow my head hurts or whatever it is. If you don't upvote you are a square

**Comment by dunsel-hordon** [comment]
why the downvotes its not like any of you will ever interact with this item?

**Comment by SonixApache** [comment]
make it a subtype with a damage var so admemes can mess with it if they want

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -493,10 +493,69 @@
...
 
+/obj/item/clothing/head/dunce_cap/OnMobLife(var/mob/living/carbon/human/dunce)
+	..()
+	if(!istype(dunce))
+		return
+	if((dunce.get_item_by_slot(slot_head) == src))
+		if(dunce.getBrainLoss() < 20)
+			dunce.adjustBrainLoss(1)
+//Brain-damaged copypaste but this is thematic
+	if(prob(1))
```
Comment (DamianX) [inline code/modules/clothing/head/misc.dm]: Absolutely unacceptable copypaste, put brain damage lines in a proc and call it instead.

**Comment by GreatBigFailure** [comment]
People are freaking out of nothing. Brain damage isn't hard to fix and the hat is rare. Not wanting to make a bit of teasing with the hat become more mean spirited is valid criticism though.
Make it remove the accumulated brain damage when you take it off, or only apply the brain damage if they've worn it long enough.

**Comment by DamianX** [comment]
I don't like it because dunce caps are pretty much admin-only and I don't want to give the guy who I'm trying to bully the opportunity to give other people brain damage, or even brain damage the guy himself in the first place. I have other buttons for that. I wouldn't be as opposed to a subtype that deals brain damage but I still wouldn't see the point.

**Comment by dunsel-hordon** [comment]
i thought barry said the brain damage was removed when it was taken off

**Comment by Kammerjunk** [comment]
dumb escalation bait for an item that already has a purpose

**Comment by boy2mantwicethefam** [comment]
@dunsel-hordon I didn't say that, but since you commissioned it I can change it to do that

**Comment by boy2mantwicethefam** [comment]
New changes for the dunce cap as per commissioner requests:
- No longer causes brain damage
- Will only make you say brain damage lines

Please re-review the PR and change your votes accordingly if you feel satisfied/unsatisfied about the change

**Comment by Kammerjunk** [comment]
you should probably read the file again, it's rolling for a line twice

**Comment by boy2mantwicethefam** [comment]
I thought I removed that, thanks

**Comment by PrimeDSS13** [comment]
Eh, people are less likely to wear it once given to them if they have to yell dunce lines all the time. I think I like it more as just a hat. Maybe add a version that doesn't make you yell the lines as well?

**Comment by dunsel-hordon** [comment]
@DamianX How dare you close this. It has more thumbs up than down. It should be merged!

**Comment by DamianX** [comment]
cringe

**Comment by dunsel-hordon** [comment]
THE EMOJIOCRACY HAS SPOKEN!

**Comment by Kammerjunk** [comment]
agree with "cringe" but code maintainers aren't feature administrators

**Pull request closed without merging**